server/models/user.js

```diff
@@ -19,6 +19,12 @@ const UserSchema = new Schema({
     type: String,
     required: true,
   },
+  friendIds: [
+    {
+      type: String,
+      required: true,
```

Comment:
This should not be type String, instead you can make the type ObjectId

Reply:
I think that having it as a `String` may be easier because it looks like `ObjectId` is returning an object. This makes it harder to parse from an array.

Maybe an alternative is to keep it as a `String` and make `friendIds` a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) so it can have a constant lookup. But I haven't found a valid way to do that right on the schema construction though. 


I think our best bet is to keep it as a `String`. What do you think?